test/runtests.jl

```diff
@@ -79,8 +86,8 @@ a[1] = Gray(0)
 a[1] = 1
 @test a[1] === Gray(1)
 
-@test @inferred(ccolor(RGB,  RGB))  == RGB
-@test @inferred(ccolor(Gray, Gray)) == Gray
+# @test @inferred(ccolor(RGB,  RGB))  == RGB
+# @test @inferred(ccolor(Gray, Gray)) == Gray
```

Comment:
Actually, I see you already did that. Maybe look for calls to parameter_upper_bound then. Those get the bounds not the typevar, so you may have to manually re-introduce that.